### ISSUE TITLE
Error when using skype.connect();

### ISSUE THREAD

**Issue by MrPowerGamerBR** [Delthas/JavaSkype#8]
```
An error occured while connecting...
fr.delthas.skype.ParseException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
	at fr.delthas.skype.WebConnector.updateContacts(WebConnector.java:119)
	at fr.delthas.skype.WebConnector.start(WebConnector.java:38)
	at fr.delthas.skype.Skype.connect(Skype.java:90)
	at fr.delthas.skype.Skype.connect(Skype.java:66)
	at com.mrpowergamerbr.skykaty.SkyKatyApp.main(SkyKatyApp.java:19)
Caused by: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
	at org.json.JSONArray.<init>(JSONArray.java:106)
	at org.json.JSONArray.<init>(JSONArray.java:145)
	at fr.delthas.skype.WebConnector.updateContacts(WebConnector.java:113)
	... 4 more
```

**Comment by MrPowerGamerBR** [comment]
Also, the default timeout is way too low, 90% of the times I got "Read timed out" when trying to login.

**Comment by MrPowerGamerBR** [comment]
@Delthas Adding ".timeout(0)" on every Jsoup connection fixes the Timeout issue (this disable the Jsoup timeout, the default 5 (?) is way too low if a account has a lot of contacts).

Also, I'm not getting the first issue anymore after you pushed this commit https://github.com/Delthas/JavaSkype/commit/626f847c7c999c75d1037affc5d828cecfe4818f , so should I close this issue or just change the title of it to "Change the default timeout"?

**Comment by Delthas** [comment]
I don't know why there was often a timeout error on login, it happened to me too.
Anyway this seems to be fixed since 626f847 indeed. I have added longer timeouts just in case.

**Comment by MrPowerGamerBR** [comment]
@Delthas (Maybe you already know this, but...)

Now I'm not facing timeout issues, thanks for fixing!
